src/backend/distributed/commands/alter_table.c

```diff
@@ -570,8 +570,11 @@ ConvertTable(TableConversionState *con)
 	char *newAccessMethod = con->accessMethod ? con->accessMethod :
 							con->originalAccessMethod;
 	IncludeSequenceDefaults includeSequenceDefaults = NEXTVAL_SEQUENCE_DEFAULTS;
+	IncludeIdentities includeIdentityDefaults = INCLUDE_IDENTITY;
```

Comment:
Since you changed the type name you can also change the variable names to includeIdentities from includeIdentityDefaults